galaxy.yml

```diff
@@ -1,10 +1,11 @@
-namespace: "pulp"
-name: "pulp"
+namespace: "mdellweg"
+name: "squeezer"
 authors:
   - "Matthias Dellweg <2500@gmx.de>"
+  - "Timo Funke <timo.funke@biotronik.com>"
```

Comment:
<3
Could you change it to `timoses@msn.com` please?

Reply:
```suggestion
  - "Timo Funke <timoses@msn.com>"
```